src/System.Management.Automation/resources/ParserStrings.resx

```diff
@@ -1316,6 +1316,9 @@ ModuleVersion : Version of module to import. If used, ModuleName must represent
   <data name="TypeNameExpected" xml:space="preserve">
     <value>Type name expected.</value>
   </data>
+  <data name="InvalidUnderlyingType" xml:space="preserve">
+    <value>'{0}' is not a valid underlying type for enums. Expected a builtin integral type (one of byte, sbyte, short, ushort, int, uint, long or ulong)</value>
```

Comment:
integral -> underlying

Reply:
Name of the error string is already updated, and the error text clearly states that it's because of an underlying type constraint

Reply:
There is 'Expected a builtin __integral__ type'.

Reply:
No, that's quite intentional. "builtin underlying type" would be ambiguous.